clients/ui/frontend/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
I don't think we need to repeat license file in submodules? `clients/ui/frontend/LICENSE`
The entire project is licensed Apache-2.0

Reply:
You are correct! My bad. I just removed it!